components/PageHeader.tsx

```
@@ -0,0 +1,47 @@
+import { type PageHeader } from "@/types";
+import React from "react";
+import PurpleIcon from "./icons/PurpleIcon";
+import { Search } from "lucide-react";
+import { Input } from "./ui/input";
+
+const PageHeader = ({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'PageHeader'. Consider to delete it or rename it.

'PageHeader' is defined here:


(lint/suspicious/noRedeclare)